### PR TITLE
WIP pocket-portis integration

### PR DESCRIPTION
**Pull request by luyzdeleon**
WIP Draft integration of the `pocket-js-core` package for the Portis `web-sdk`.

### CODE REVIEW

File: packages/portis-web3/src/index.ts [hunk 1]
```diff
@@ -43,6 +46,15 @@ export default class Portis {
     };
     this.widget = this._initWidget();
     this.provider = this._initProvider();
+
+    // Setup pocket
+    this.pocket = new PocketJSCore.Pocket({
+      devID: 'devIdHere',
```
Comment (radotzki) [inline packages/portis-web3/src/index.ts:52]: You can extend `INetwork` and add `pocketDevID`

Comment (radotzki) [inline packages/portis-web3/src/index.ts:52]:
```suggestion
      devID: this.config.network.pocketDevID,
```

File: packages/portis-web3/src/index.ts [hunk 2]
```diff
@@ -43,6 +46,15 @@ export default class Portis {
     };
     this.widget = this._initWidget();
     this.provider = this._initProvider();
+
+    // Setup pocket
+    this.pocket = new PocketJSCore.Pocket({
```
Comment (radotzki) [inline packages/portis-web3/src/index.ts:51]: Init `pocket` and `pocketProvider` only if `this.config.network.pocketDevID` exists

**Review by radotzki — CHANGES_REQUESTED** [review]

**Comment by radotzki** [comment]
@luyzdeleon please open a PR on our repo, so we will be able to iterate on it together. It looks good!

**Comment by luyzdeleon** [comment]
> @luyzdeleon please open a PR on our repo, so we will be able to iterate on it together. It looks good!

@radotzki Will definitely do. The purpose of this draft PR was to show you a mock of how the implementation would look like and what would be the impact scope for the Portis Web SDK.

**Pull request closed without merging**